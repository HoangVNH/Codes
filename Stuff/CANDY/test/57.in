10
67
VXXDXXDDDXDXVXDDDXDVVVXXDDXVVVXXVDVDDVXXVXVXXDVXDXVXDXXXVXDXDXXVXVD
86
XDVVVVVVDVDDDXVVVXXVXDVXVDVXDVXDDVXXDXVXVVXXXDVXXXDDDXDVDXDDVXXDVXXDXXXXDDDXVVDVXDXXXD
95
DDVXVVDVDXDXDDXDDXDVVDXVXDVVXVVVXXVVXXVVDXDDXDDVVXVXDVDVDDVVDVVDXVDXVDVVDVDXDVDXXDVVVXDDVDXVVXV
61
VXDDDVXXVXVDVXXDDXXDDXDXVVXXXVDVDVXVVXVVVXVDXDVVDXVVDXXDDVVXX
62
XXVVVVVDDVVVXDVVXDDDVVDDVVXVXDXXVDXVDXXXDDXXVVDDVVVVXVDDVXDDDV
88
DDXDXDXXXXVXVVVDVDXVDVDDXDXDVDDDXDDDDVDDVDVDXVVDVVXVVVVVVVVVXVDDDVDXXDVDXDDDXDXVXXDDXDXX
52
VVDDVVDDDXDXDXVVDDVXDDDVVXDVDXVXXXVDVVDDDDDXDXDDDVVD
74
DXDXDXDDDXDXDXDDXXVDXVVDXXVXVVXDXVXVDXXVXXDVVVVXDXXDVXXDDVXDXDDXXVDDVDDDXV
59
DDDVDVVDXVVDVDXDVXXDVVVDVVVXXDVDXXVVDXVDDVDVXDXDVDXDXVDXVDX
65
VDVXXVDVXXDVVXXXVXDDDDDDVXVXVXXXXXVXVVXXDDVVDXXXDVDDXVDVDVXVVXXXV
