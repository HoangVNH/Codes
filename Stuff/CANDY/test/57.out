66
85
94
60
60
86
51
73
58
64
